configure

```diff
@@ -1,4 +1,14 @@
-#!/usr/bin/env python
+#!/bin/sh
+
+# Locate python2 interpreter and re-execute the script.  Note that the
+# mix of single and double quotes is intentional, as is the fact that
+# the ] goes on a new line.
+_=[ 'exec' '/bin/sh' '-c' '''
+which python2.7 >/dev/null && exec python2.7 "$0" "$@"
+which python2 >/dev/null && exec python2 "$0" "$@"
+exec python "$0" "$@"
+''' "$0" "$@"
+]
 
```

Comment:
Maybe add a `del _` or `_ = None` so that the `_` variable doesn't propagate through the script. I doubt it'll conflict with anything (or cause a performance hit), but it would at least clarify that the python `_` variable creation is a byproduct of the shell script.